packages/usuario/src/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @description Export all features and business rules from the package
+ */
+
+// Domains
+const { findQuery, findOneQuery, createQuery } = require('./domains');
+
+// Services
+const { find, findOne, create } = require('./useCases');
+
+// Models
+const usuarioModel = require('./drivers/db/models/usuario');
+
+module.exports = {
+  methods: {
+    find: find(findQuery),
+    findOne: findOne(findOneQuery),
+    create: create(createQuery),
+  },
+  models: {
+    usuario: usuarioModel,
+  },
+};
```

Comment:
Could we add a new line here? You can read why it is needed [here](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)